packages/app/src/state/cortex-config/sagas/cortexConfigSubscription.ts

```diff
@@ -113,7 +114,12 @@ export function cortexConfigSubscriptionEventChannel(): EventChannel<Actions> {
             emitter(actions.setCortexRuntimeConfig(res.data));
           })
           .catch(err => {
-            emitter(actions.setCortexConfigError(err.toString()));
+            if (ServerError.isInstance(err.response.data)) {
```

Comment:
client handling of errors: 1) check if the error came from the server and extract the detailed message, 2) fallback to a generic error, usually in the case of a network connection problem.